lte/gateway/c/oai/tasks/sgw_s8/sgw_s8_handlers.c

```diff
@@ -505,6 +507,31 @@ void sgw_s8_handle_create_session_response(
         session_rsp_p->cause, session_rsp_p->context_teid);
     sgw_remove_sgw_bearer_context_information(
         sgw_state, session_rsp_p->context_teid, imsi64);
+  } else {
```

Comment:
is it possible to put this into a function like `update_subscriber_sgw_c_teids_to_directoryd` (or some shorter name)

Reply:
Sure